src/style.css

```diff
@@ -0,0 +1,158 @@
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Regular.ttf);
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Medium.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Bold.ttf);
+  font-weight: 700;
+  font-style: normal;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  --color-buy: #00acdc;
+  --color-text: #060b35;
+  --color-transparent: #616070;
+}
+
+.card {
```

Comment:
Follow the checklist, please 
<img width="938" alt="image" src="https://user-images.githubusercontent.com/99403198/219865404-489a22ce-c2d0-4edd-8b13-5202725def88.png">